web3/gas_strategies/time_based.py

```diff
@@ -73,7 +73,7 @@ def _compute_probabilities(miner_data, wait_blocks, sample_size):
         min_gas_price = miner_data_by_price[idx].min_gas_price
         num_blocks_accepting_price = sum(m.num_blocks for m in miner_data_by_price[idx:])
         inv_prob_per_block = (sample_size - num_blocks_accepting_price) / sample_size
-        probability_accepted = 1 - inv_prob_per_block ** wait_blocks
+        probability_accepted = (1 - inv_prob_per_block) ** wait_blocks
         yield Probability(min_gas_price, probability_accepted)
```

Comment:
Ok, now I'm more confused.

If 1 block accepted the given price in a sample of 4, then the new code says this, which looks wrong:
```py
inv_prob_per_block = 3/4
probability_accepted = (1/4) ^ wait_blocks
```
The old code made more sense to me:
```py
inv_prob_per_block = 3/4
probability_accepted = 1 - (3/4 ^ wait_blocks)
```
Where the parenthetical probability is the chance that the gas price would never be accepted in any of the wait-blocks.

Reply:
I'm sleepy and wrong.  You're correct now that I'm looking at it.

> goes back to reviewing code.